cypress/integration/prospects/note.js

```diff
@@ -0,0 +1,39 @@
+describe('Create a prospect note', () => {
```

Comment:
I think you can  put this test inside the prospect profile file for testing (`integration/prospects/profile`)